### PR TITLE
Dev add to documentation

### PR DESCRIPTION
**Pull request by neurosutton**
Includes a couple of doc strings to summarize the build functions.
Adds a couple of notes about where the modules are sourced, based on the modular HCP gears' design.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,9 +24,9 @@ RUN apt-get update && \
     apt-key add /tmp/neurodebian_pgpkey.txt && \
     apt-get update && \
     apt-get install -y fsl-core=5.0.9-5~nd14.04+1 \
-    python3-pip && \
+    python3-pip \
```
Comment (NPann) [inline Dockerfile]: Don't you need a `&&` here ?

File: utils/args/GenericfMRISurfaceProcessingPipeline.py [hunk 2]
```diff
@@ -6,12 +6,20 @@
 import logging
 import os.path as op
 
+# Note common in available from hcp-base, once the Docker image is pulled.
```
Comment (NPann) [inline utils/args/GenericfMRISurfaceProcessingPipeline.py]:
```suggestion
# Note common is available from hcp-base, once the Docker image is pulled.
```

**Review by NPann — APPROVED** [review]
LGTM but that comment on the Dockerfile. Approving preemptively.

File: Dockerfile (hunk 1 above)
Comment (neurosutton) [reply Dockerfile]: Totally do. Thank you!

**Pull request merged**